packages/next/build/webpack/loaders/next-view-loader.ts

```diff
@@ -119,6 +119,8 @@ const nextViewLoader: webpack.LoaderDefinitionFunction<{
     export const components = {
         ${componentsCode.join(',\n')}
     };
+
+    export const webpackRequire = __webpack_require__
```

Comment:
Couldn't use `__next_rsc__` here as it would cause the virtual module to be handled as a server component

Reply:
Consider renaming it to `__next_view_webpack_require__` or `__view_webpack_require__` to differentiate from rsc webpack require?

Reply:
Done!